packages/sources/cfbenchmarks-test/src/index.ts

```diff
@@ -0,0 +1,15 @@
+import { expose, ServerInstance } from '@chainlink/external-adapter-framework'
+import { PriceAdapter } from '@chainlink/external-adapter-framework/adapter'
+import { SettingsMap } from '@chainlink/external-adapter-framework/config'
+import { customSettings } from './config'
+import { crypto } from './endpoint'
+
+export const makeAdapter = (): PriceAdapter<SettingsMap> =>
```

Comment:
The correct type for the generic param would be `typeof customSettings`, but it should be able to infer it, does 
```suggestion
export const makeAdapter = (): PriceAdapter =>
```
work?

Reply:
It does not.

Reply:
Ah, mb. Have a PR up in the repo that should fix it

Reply:
Still not working as expected with 0.10.1

![image](https://user-images.githubusercontent.com/28818476/200408953-438b0180-9893-4baf-94e4-da6b5f40f9a7.png)

Reply:
I'll pull the branch and check

Reply:
`0.10.2` should fix this, second time's the charm